src/main/java/ai/starlake/transpiler/JSQLColumResolver.java

```diff
@@ -194,6 +195,28 @@ public JdbcResultSetMetaData getResultSetMetaData(String sqlStr) throws JSQLPars
     }
   }
 
+  /**
```

Comment:
Please why do we need this method here?

Reply:
Ability to configure the underlying parser - e.g. to handle T-SQL way of quoting (square brackets).